ee/modules/110-istio/monitoring/prometheus-rules/versions.tpl

```diff
@@ -3,9 +3,9 @@
     - alert: D8IstioDeprecatedIstioVersionInstalled
       annotations:
         description: |
-          There is deprecated istio version `{{$labels.version}}` installed.
+          There is deprecated istio version {{`{{$labels.version}}`}} installed.
```

Comment:
```suggestion
          There is deprecated istio version `{{"{{$labels.version}}"}}` installed.
```
Your
```
{{`{{$labels.version}}`}}
```
results in `{{$labels.version}}` without quotations "`" in Prometheus it is being rendered to `1.13`. But we need commas around the version.
Inside "{{}}" i changed "`" to '"' for more transparency.